code.html

```diff
@@ -0,0 +1,206 @@
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#About"><p style="font-size:2vw;">About</p></a>
+          </li>
+          <!--<li class="nav-item">
+            <a class="nav-link" href="index.html#News"><p style="font-size:2vw;">News</p></a>
+          </li>-->
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#Team"><p style="font-size:2vw;">Team</p></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="index.html#Funding"><p style="font-size:2vw;">Funding</p></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="papers.html"><p style="font-size:2vw;">Papers</p></a>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="#"><p style="font-size:2vw;">Code</p></a>
+          </li>
+          <!--<li class="nav-item">
+            <a class="nav-link" href="./docs/index.html" target="_blank"><p style="font-size:2vw;">Docs</p></a>
+          </li>-->
+          <!-- <li class="nav-item">
+            <a class="nav-link" href="https://data.posydon.org" target="_blank"><p style="font-size:2vw;">Database</p></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="https://posydon.odahub.io" target="_blank"><p style="font-size:2vw;">Webapp</p></a>
+          </li> -->
+        </ul>
+      </div>
+    </nav>
+
+    <div class="container p-4">
+      <p></p>
+      <h5>Links to the latest versions of our code, population synthesis data, documentation, and associated instrument papers are provided below.</h5>
+      <p></p>
+    </div>
+    <div class="container p-4">
+        <h1>POSYDON v2.0.0</h1>
+        <p>
+          <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#v2" aria-expanded="false" aria-controls="v2">
+            links
+          </button>
+        </p>
+        <div class="collapse" id="v2">
+          <div class="card card-body">
+            <p><b>Code: </b><a href="https://github.com/POSYDON-code/POSYDON/" target="_blank">v2.0.0 (development)</a></p>
+            <p><b>Documentation: </b><a href="https://posydon.org/POSYDON/" target="_blank">Installation, tutorials, API</a></p>
+            <p><b>Population synthesis data (v2): </b>TBA</p>
+            <p><b>Population synthesis data (v1): </b><a href="https://zenodo.org/records/14205146" target="_blank">10.5281/zenodo.14205146</a> (v2.x.x-compatible)</p>
+            <p><b>Instrument Paper: </b> <a href="https://ui.adsabs.harvard.edu/abs/2024arXiv241102376A/abstract" target="_blank">Andrews et al. 2024</a></p>
+          </div>
+        </div>
+    </div>
+
+    <div class="container p-4">
+      <h1>POSYDON v1.0.4</h1>
+      <p>
+        <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#v1" aria-expanded="false" aria-controls="v1">

```

Comment:
Similar to the Papers, I think having a dropdown here is a bit awkward. I think it would look good to just have the box with all the version info directly below the "POSYDON v2.0.0", "POSYDON v1.0.4" headings.

Reply:
Responding to your comment above, I personally like this format for the data, code, etc., I think it is very clear.